conf/broker.conf

```
@@ -58,6 +58,10 @@ advertisedAddress=
 # Number of threads to use for Netty IO. Default is set to 2 * Runtime.getRuntime().availableProcessors()
 numIOThreads=
 
+# Number of threads to use for ordered executor. The ordered executor is used to operate with zookeeper,
+# such as init zookeeper client, get namespace policies from zookeeper etc. It also used to split bundle. Default is 8
+numOrderedExecutorThreads=
```

If we have a fixed default, we should have the value here, unless the number is computed based on the available processors (which is in most cases a better approach)

```suggestion
numOrderedExecutorThreads=8
```